ip_localities.py

```diff
@@ -0,0 +1,79 @@
+from arcgis import GIS
+import pandas as pd
+import math
+
+
+def meters_to_long(x):
+    # Converts longitude point from Spherical Mercator EPSG:900913 to lat/lon in WGS84 Datum
+    origin_shift = 2 * math.pi * 6378137 / 2.0
+    long = (x / origin_shift) * 180.0
+    return long
+
+
+def meters_to_lat(y):
```

Comment:
Same set of comments as before. 

In fact, given the similarities you could maybe combine these guys. Pass in whether it's a longitude or latitude and then have the last one that does the fancy math be an if statement.

Reply:
If you think that would be cleaner, will do!